extensions/amp-story/1.0/amp-story.js

```diff
@@ -946,7 +946,21 @@ export class AmpStory extends AMP.BaseElement {
         );
 
         if (shouldReOpenAttachmentForPageId === this.activePage_.element.id) {
-          this.activePage_.openAttachment(false /** shouldAnimate */);
+          const attachmentEl = this.activePage_.element.querySelector(
+            'amp-story-page-attachment, amp-story-page-outlink, amp-story-shopping-attachment'
+          );
+
+          if (!attachmentEl) {
```

Comment:
This return is preventing the code below from running, you can instead do `if (attachmentEl) {whenUpgraded...}`